spec/models/spree/user_spec.rb

```diff
@@ -24,6 +24,18 @@
         }.to raise_error ActiveRecord::RecordInvalid, "Validation failed: #{u2.email} is not permitted to own any more enterprises (limit is 1)."
       end
     end
+
+    describe "group ownership" do
+      let(:u1) { create(:user) }
+      let(:u2) { create(:user) }
+      let!(:g1) { create(:enterprise_group, owner: u1) }
+      let!(:g2) { create(:enterprise_group, owner: u1) }
```

Comment:
Maybe include a group owned by someone else to show that it's only including owned groups.